packages/options/legacy/class-jetpack-options.php

```diff
@@ -558,6 +558,9 @@ public static function get_all_wp_options() {
 			'disabled_likes',
 			'disabled_reblogs',
 			'jetpack_comments_likes_enabled',
+			'wp_mobile_excerpt',
```

Comment:
I think we can still get rid of these. This function is called when JP is deleted or having the options reset, but they would already have been deleted in `plugin_upgrade`

Reply:
Done in eb9b8991b83569107bb63538c1f3c9092268bd4c